modules/oe_content_publication/config/install/core.entity_view_display.node.oe_publication.default.yml

```diff
@@ -2,23 +2,82 @@ langcode: en
 status: true
```

Comment:
We changed this configuration but we don't have an update path for it.

Reply:
Do we want to update existing node view on already created sites? Won't we break any custom configuration?

Reply:
The same will happen for form displays though, any existing customizations will be lost, lets check this with the rest of the team before committing.

Reply:
Since this is the default view mode and existing sites should use the full mode anyways (thats what we do on oe_theme) we can safely update it.

Reply:
This is fixed.

Reply:
We have not updated any core.entity_view.* on oe_content before.